conf/defaults.yaml

```diff
@@ -263,7 +263,7 @@ topology.state.checkpoint.interval.ms: 1000
 # topology priority describing the importance of the topology in decreasing importance starting from 0 (i.e. 0 is the highest priority and the priority importance decreases as the priority number increases).
 # Recommended range of 0-29 but no hard limit set.
 topology.priority: 29
-topology.component.resources.onheap.memory.mb: 128.0
+topology.component.resources.onheap.memory.mb: 256.0
```

Comment:
Should we have a separate default value for system components? Otherwise, users' topologies might use more memory than their previous impression. E.g., a WordCount example might need 6GB to start. Since this is an important setting, @revans2 , what is your opinion?